arborator/main.py

```diff
@@ -77,6 +77,9 @@
 CLUSTER_METHOD_LONG = "--" + CLUSTER_METHOD_KEY
 CLUSTER_METHOD_SHORT = "-e"
 
+TREE_DISTANCES_KEY = "tree_distances"
```

Comment:
`tree_distances` instead of `tree-distances` (like in GAS) because all the command line parameters in Arborator use underscores.